styles/_search_results.scss

```diff
@@ -80,6 +92,34 @@
       }
     }
   }
+
+  .meta-group {
+    display: flex;
+    align-items: center;
+    clear: right;
```

Comment:
Same here. Is this `clear` rule needed?

Reply:
I used `float: right` to align # of votes to the right of article titles. If I don't clear that the breadcrumbs in search results will not be displayed correctly.

Reply:
Got it! I missed that float rule 👍